content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst

```diff
@@ -55,77 +55,61 @@ A :guilabel:`Receipt` smart button will appear in the top right, and the receipt
 with the purchase order. Clicking on the :guilabel:`Receipt` smart button will show the receipt
 order.
 
-.. image:: receipts_three_steps/three-step-purchase-receipt.png
+.. image:: receipts_three_steps/receipts-three-steps-smart-button.png
    :align: center
    :alt: After confirming a purchase order, a Receipt smart button will appear.
 
 Process a receipt
 -----------------
 
-One receipt and two internal transfers (one transfer to quality, and a subsequent transfer to stock)
-will be created once the purchase order is confirmed. To view these transfers, go to
-:menuselection:`Inventory --> Operations --> Transfers`.
+Once the purchase order (PO) is confirmed, a receipt (`WH/IN`) operation is generated and ready to
+process.
 
-.. image:: receipts_three_steps/three-step-transfers.png
-   :align: center
-   :alt: The status of the three receipt transfers will show which operation is ready and which ones
-         are waiting another operation.
-
-The status of the receipt transferring the product to the input location will be :guilabel:`Ready`,
-since the receipt must be processed before any other operation can occur. The status of the two
-internal transfers will be :guilabel:`Waiting Another Operation`, since the transfers cannot be
-processed until the linked step before each transfer is completed.
-
-The status of the first internal transfer to *quality* will only change to :guilabel:`Ready` when
-the receipt has been marked :guilabel:`Done`. The status for the second internal transfer to *stock*
-will be marked :guilabel:`Ready` only after the transfer to quality has been marked
-:guilabel:`Done`.
+The receipt can be confirmed from the original purchase order form, or it can be accessed by
+navigating to the :menuselection:`Inventory app`, and locating the :guilabel:`Receipts` task card.
 
-The receipt can also be found in the :menuselection:`Inventory` application. In the
-:guilabel:`Overview` dashboard, click the :guilabel:`1 To Process` smart button in the
-:guilabel:`Receipts` kanban card.
-
-.. image:: receipts_three_steps/three-step-receive-kanban.png
-   :align: center
-   :alt: One Receipt ready to process in the Inventory Overview kanban view.
+Click the :guilabel:`# To Process` button to reveal all incoming receipts to process. Click the
+receipt associated with the previous purchase order.
 
-Click on the receipt associated with the purchase order, then click :guilabel:`Validate` to complete
-the receipt and move the product to the :guilabel:`Input Location`.
+Click :guilabel:`Validate` to validate the receipt, and move the product to the destination
+location, :guilabel:`WH/Input`.
 
-.. image:: receipts_three_steps/validate-three-step-receipt.png
+.. image:: receipts_three_steps/receipts-three-steps-receipt-form.png
```

Comment:
This screenshot is a bit too zoomed in and removes the context of the surrounding UI elements.

Reply:
Screenshot replaced with new zoomed-out one that shows navigational UI and highlights key areas. Suggestion implemented ✅